docs/_sources/meta.rst.txt

```diff
@@ -356,7 +356,7 @@ Normally Python packages should use this line:
 .. code-block:: yaml
 
     build:
-      script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+      script: "{{ PYTHON }} -m pip install . -vvv"
```

Comment:
This is unfortunately an autogenerated file and not the source itself. So this didn't show up in the docs. Fixing with PR ( https://github.com/conda-forge/conda-forge.github.io/pull/659 ).